bin/create_folders.py

```diff
@@ -70,22 +71,21 @@ def update_permissions(
         syn.setPermissions(subfolder, principalId=principal_id, accessType=access_type)
 
 
-def build_update_subfolders(
+def create_folders(
     project_name: str,
     submission_id: str,
     build_or_update: str,
     subfolders: List[str] = ["workflow_logs", "predictions"],
     only_admins: str = "predictions",
-    parent_folder: str = "Logs",
-):
+    root_folder_name: str = "Logs",
+) -> None:
     """
-    This function can either build/rebuild a set of log subfolders under
-    a participant folder, or update an existing participant folder with
-    new ancilliary files.
+    This function can either create or re-create a root folder and set of subfolders to
+    store Challenge output files for Challenge participants and organizers.
 
     The current Challenge Folder structure is as follows:
 
-    Parent-Folder/
+    Root-Folder/
```

Comment:
Out of scope, but I changed the language from "parent folder" to "root folder" because the words "project" and "parent" can get mixed up and make the code look more confusing